packages/oauth/src/stores/file-store.ts

```diff
@@ -0,0 +1,116 @@
+import { Installation, InstallationStore, InstallationQuery } from '../index';
+import { Logger } from '../logger';
+import fs from 'fs';
+import path from 'path';
+import { homedir } from 'os';
+
+export interface FileInstallationOptions {
+  baseDir?: string;
+  historicalDataEnabled?: boolean;
+  clientId?: string;
+}
+
+export class FileInstallationStore implements InstallationStore {
+
+  private baseDir: string;
+  private historicalDataEnabled: boolean;
+
+  constructor({
+    baseDir = `${homedir()}/.bolt-app-installation`,
+    clientId,
+    historicalDataEnabled = true,
+  }: FileInstallationOptions = {}) {
+    this.baseDir = clientId !== undefined ? `${baseDir}/${clientId}` : baseDir;
+    this.historicalDataEnabled = historicalDataEnabled;
+  }
+
+  public async storeInstallation(installation: Installation, logger?: Logger): Promise<void> {
+    const { enterprise, team, user } = installation;
+    const installationData = JSON.stringify(installation);
+    const installationDir = this.getInstallationDir(enterprise?.id, team?.id);
+
+    if (logger !== undefined) { logger.warn('Storing installation in FileInstallationStore'); }
```

Comment:
To me, info level would be more reasonable here as this is not an error or failure. Also, having useful data like enterprise_id / team_id / user_id in the message would be a good addition (needless to say, we should not have tokens / webhook URLs). Just having this message does not add so much value.

Reply:
All good points. I did carry this over from the memory store, but did away with the true 'warning' part of it (recommending that they use a _real_ store for production purposes).

If using the `FileInstallationStore` is not something we'd recommend for production, it may be worth adding that back into the message and keeping it as a warning to remind them. Thoughts?

Reply:
Perhaps something like the following:

```
if (logger !== undefined) {
  logger.info(`Deleting installations in ${installationDir} using the following query: ${query}`);
  logger.warn('FileInstallationStore is not intended for production purposes.');
}
```

Reply:
How about having the warning log only for this store instance's initialization? I think repeatedly seeing this warning when local development is not a great developer experience.